integration/use-cases/operator-deployment.js

```diff
@@ -22,13 +24,21 @@ test("Deploys an Operator", async () => {
     await expect(page).toClick("cds-button", { text: "Deploy" });
   });
 
-  // Deploy the Operator
-  await expect(page).toClick("cds-button", { text: "Deploy" });
+  const isAlreadyDeployed = await page.evaluate(
+    () => document.querySelector("cds-button[disabled]") !== null,
+  );
 
-  await utils.retryAndRefresh(page, 2, async () => {
-    // The CSV takes a bit to get populated
-    await expect(page).toMatch("Installed");
-  });
+  if (!isAlreadyDeployed) {
+    // Deploy the Operator
+    await expect(page).toClick("cds-button", { text: "Deploy" });
+
+    await utils.retryAndRefresh(page, 4, async () => {
+      // The CSV takes a bit to get populated
+      await expect(page).toMatch("Installed", { timeout: 10000 });
+    });
+  } else {
+    console.log("Warning: the operator has already been deployed");
+  }
```

Comment:
not sure if you already replied, why the `isAlreadyDeployed`?

Reply:
Yep (sorry for dropping the former PR, but I wanted to split the changes for ease the review process)
Check it at https://github.com/kubeapps/kubeapps/pull/2610#discussion_r602470780
Since uninstalling the operator is not that easy, I left this `if` for those cases where we are executing the tests locally. I don't mind removing it if you want, though.

Reply:
it should be fine (if for some reason this is bypassing the operator installation, it should fail when creating an instance).